staging/src/k8s.io/dynamic-resource-allocation/go.mod

```diff
@@ -82,6 +84,7 @@ replace (
 	k8s.io/apiserver => ../apiserver
 	k8s.io/client-go => ../client-go
 	k8s.io/component-base => ../component-base
+	k8s.io/component-helpers => ../component-helpers
```

Comment:
q: Does the update-vendor script takes care of adding this replace automatically, or its a manual change?

Reply:
update script adds this